packages/atomic/src/components/commerce/product-template-components/atomic-product-description/e2e/atomic-product-description.e2e.ts

```diff
@@ -0,0 +1,128 @@
+import {test, expect} from './fixture';
+
+test.describe('atomic-product-description', async () => {
+  test.beforeEach(async ({page, productDescription}) => {
+    await page.setViewportSize({width: 375, height: 667});
+    await productDescription.load();
+    await productDescription.hydrated.first().waitFor();
+  });
+
+  test('should be accessible', async ({makeAxeBuilder}) => {
+    const accessibilityResults = await makeAxeBuilder().analyze();
+    expect(accessibilityResults.violations.length).toEqual(0);
+  });
+
+  const fields: Array<'ec_description' | 'ec_shortdesc'> = [
+    'ec_description',
+    'ec_shortdesc',
+  ];
+
+  fields.forEach((field) => {
+    test(`should render description for ${field} field`, async ({
+      productDescription,
+    }) => {
+      await productDescription.load({args: {field}});
+      await productDescription.hydrated.first().waitFor();
+
+      expect(productDescription.textContent.first()).toBeVisible();
+    });
+  });
+
+  test.describe('when description is truncated', async () => {
```

Comment:
Same here, we could test that passing an invalid value throws a validation error.